phpBB/styles/prosilver/template/faq_body.html

```diff
@@ -36,7 +36,7 @@ <h2 class="faq-title">{faq_block.BLOCK_TITLE}</h2>
 					<dd>{faq_block.faq_row.FAQ_ANSWER}</dd>
 				</dl>
 				<a href="#faqlinks" class="top">
-					{{ Icon('iconify', 'fa:chevron-circle-up', lang('BACK_TO_TOP'), false) }}
+					{{ Icon('font', 'circle-chevron-up', lang('BACK_TO_TOP'), false) }}
```

Comment:
You dont add `fas` here in the 5 parameter?

Reply:
is `fas` always needed? I was unclear about it, and if it is always needed, then why isn't it made into a default for that parameter so it doesn't always need to be added?

Reply:
I think fas is used when you don't specify anything, so it can be removed

Reply:
Yes, fas (or fa-solid) is the default one. If you're not using the default one you have to explicitly define it.